Graphs/Area/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+var AreaChart = require('react-d3').AreaChart;
```

Comment:
Rather than require the entire react-d3 library, find the module in the library that provides the `AreaChart` and require only that.  Check out how the tux/Router works to see how to do this.